src/cloud-api-adaptor/podvm/Dockerfile.podvm.fedora

```diff
@@ -0,0 +1,10 @@
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Place pod vm image built via mkosi
+#
+
+FROM scratch
+
+COPY build/podvm-s390x.qcow2 /
```

Comment:
Should this have an ARCH check ?

Reply:
Currently we only place s390x qcow2 image in an OCI image and push it to quay.io registry: `quay.io/confidential-containers/podvm-generic-fedora-s390x`

I just found the x86 image via mkosi is built in another workflow, but it doesn't push image to quay.io: https://github.com/confidential-containers/cloud-api-adaptor/actions/workflows/podvm_mkosi.yaml

So no check here, but we can improve it if we need place x86 image in OCI image and push to quay.io in future.

Reply:
This DockerFile is generic. So imho it should follow the pattern used elsewhere in the project to avoid inconsistencies and issues later.

Reply:
OK. I try to add a ARG ARCH in the Dockerfile, referring to `Dockerfile.podvm` and `Dockerfile.podvm.rhel`